app/regen/cmd/root.go

```diff
@@ -79,6 +81,9 @@ func Execute(rootCmd *cobra.Command) error {
 	ctx = context.WithValue(ctx, client.ClientContextKey, &client.Context{})
 	ctx = context.WithValue(ctx, server.ServerContextKey, server.NewDefaultContext())
 
+	rootCmd.PersistentFlags().String(flags.FlagLogLevel, zerolog.InfoLevel.String(), "The logging level (trace|debug|info|warn|error|fatal|panic)")
+	rootCmd.PersistentFlags().String(flags.FlagLogFormat, tmcfg.LogFormatPlain, "The logging format (json|plain)")
```

Comment:
The reason why we cannot just use `github.com/cosmos/cosmos-sdk/server/cmd` is because of the REGEN env prefix, is that right?

Reply:
Yes, we cannot use REGEN env prefix.